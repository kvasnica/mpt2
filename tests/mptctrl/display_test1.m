function display_test1

load ctrl1
ctrl
