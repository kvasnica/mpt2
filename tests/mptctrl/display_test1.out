
ctrl=

Explicit MPC controller defined over 9 regions in 2D            
                                                                
   Controlled system: LTI system, 2 states, 1 input, 2 outputs  
   Control objective: horizon 1 optimal control problem (2-norm)
          Regulation: towards origin                            
          Invariance: the closed-loop system is invariant       
           Stability: the closed-loop system is stable          

 Type 'struct(ctrl)' to see all fields.
 Fields can be accessed directly, e.g. type 'ctrl.Pn' to get the controller partition.